### PR TITLE
Gm/players middleware

### PR DESCRIPTION
**Pull request by gabinho5**
## Summary of Changes
Added middleware for players and new players reducer
Added DraftStack for all of our different draft screens
Added PlayersScreen to start off the players selection process
Fixed react native debugger (devtools)

### CODE REVIEW

**Comment by ajsun** [comment]
👻 This branch as been deployed to:
            (https://expo.io/@poppop/nutmeg?release-channel=gm-players-middleware)

File: src/redux/store.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import allSagas from './sagas';
 const sagaMiddleware = createSagaMiddleware();
 
 // Setting up redux devtool enhancers
-const middleware = applyMiddleware(sagaMiddleware);
+const middleware = applyMiddleware(sagaMiddleware, logger);
```
Comment (ajsun) [inline src/redux/store.js]: wow this is life changing

Comment (gabinho5) [reply src/redux/store.js]: @ajsun You like that?

Comment (ajsun) [reply src/redux/store.js]: go ahead and remove the logger

**Comment by ajsun** [comment]
👻 This branch as been deployed to:
            (https://expo.io/@poppop/nutmeg?release-channel=gm-players-middleware)

**Pull request merged**